src/types/options/qualityRequestOptions.ts

```diff
@@ -0,0 +1,102 @@
+import { CoreServiceUrl, OSMType, PoiGroup, TravelMode, TravelTimeFactors } from '..';
+
+export interface QualityRequestOptions {
+  /** Criterion definitions
+   * For each criterion, a key must be set to be able to identify the different criteria in the response */
+  [criteriaId: string]: (PointOfInterestCriterion | StatisticsCriterion | GravitationalCriterion)
+}
+
+
+interface BaseCriterion {
+  /** This is the service URL to which the Targomo Reachability requests are dispatched. The endpoint has to correspond to the addresses
```

Comment:
I would replace 'Targomo Reachability requests' by 'Targomo Routing requests' (not sure about that term though) to be more generic. poi service is using reachability requests but not statistics service (espcially when you do gravitation request).

Reply:
but I guess this is taken from the docs.